### ISSUE TITLE
Spike: Make sure logout does not pose accessibility issues

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#783]
Part of https://github.com/hypothesis/product-backlog/issues/753.

The accessibility report says we fail this WCAG guideline:

> For login sessions that expire in less than 20 hours, users require the ability to disable, adjust,
or extend the time limit before it is encountered. Before expiry, warn the user and give at least
20 seconds to extend the time limit with a simple action (for example, "press the space bar"),
with the user being allowed to extend the time limit at least ten times.

I have never been logged out while using H so I'm not sure if this is true! We need to determine:

- How long will a user stay logged in to Hypothesis if they are on the same web page? Is it less than 20 hours?
- What happens when the user is logged out? Does the client silently refresh? 

Note: This is a research issue to determine if we are compliant in this area. If our login lasts more than 20 hours, no development work is needed.

**Comment by ajpeddakotla** [comment]
Closed in favor of #1085
